trpcultivate_phenoshare/src/Plugin/TripalImporter/TripalCultivatePhenoshareImporter.php

```diff
@@ -42,6 +48,8 @@
  */
 class TripalCultivatePhenoshareImporter extends ChadoImporterBase implements ContainerFactoryPluginInterface {
```

Comment:
Should we rename this class to use an uppercase "S" in "share"?
```suggestion
class TripalCultivatePhenoShareImporter extends ChadoImporterBase implements ContainerFactoryPluginInterface {
```

Reply:
I agree with this rename! It will also require renaming the file to match.

Reply:
Renamed to TripalCultivatePhenoShareImporter